src/utils/tokens.ts

```diff
@@ -68,16 +68,18 @@ export async function getPoolTokensV3(
 ): Promise<Address[]> {
     try {
         const chainId = await client.getChainId();
-        const vaultV3 = getContract({
-            abi: vaultExtensionAbi_V3,
-            address: VAULT_V3[chainId],
+        const vaultExplorer = getContract({
+            abi: vaultExplorerAbi_V3,
+            address: VAULT_EXPLORER_V3[chainId],
             client: client,
         });
-        return (await vaultV3.read.getPoolTokens([poolAddress])) as Address[];
+        return (await vaultExplorer.read.getPoolTokens([
+            poolAddress,
+        ])) as Address[];
     } catch (e) {
         console.warn(e);
         throw new Error(
-            `Error: Unable to get pool tokens using this pool address: ${poolAddress}`,
+            `Unable to get pool tokens using this pool address: ${poolAddress}`,
```

Comment:
I don't understand why this stopped working, but I'm guessing vault explorer will work?

Will vault explorer be deployed for live networks?

Reply:
I also don't understand why vault call would fail here and vaultExplorer would succeed 🤔